src/index.js

```diff
@@ -15,27 +15,37 @@ chrome.browserAction.setPopup({ popup: 'settings.html' })
 
 const frameState = {
   connected: false,
-  availableChains: []
+  availableChains: [],
+  currentChain: ''
 }
 
 function setChains (chains) {
-  frameState.availableChains = chains.map(c => parseInt(c))
+  frameState.availableChains = chains
```

Comment:
does this still work the same way with hex values instead of numbers?

Reply:
These chains will be `{ id, name }` objects (will add a logo src here too) and the `id` is hex.

Reply:
👍🏻